content/en/guides/models/track/log/distributed-training.md

```diff
@@ -100,6 +100,51 @@ Explore the W&B App UI to view an [example dashboard](https://wandb.ai/ayush-tha
 
 The preceding image demonstrates the W&B App UI dashboard. On the sidebar we see two experiments. One labeled 'null' and a second (bound by a yellow box) called 'DPP'. If you expand the group (select the Group dropdown) you will see the W&B Runs that are associated to that experiment.
 
+### Organize distributed runs with `job_type`
+
+When running distributed training with multiple processes, you can use `job_type` to distinguish between different types of nodes and create cleaner workspaces. This approach helps you organize your runs and filter out noise from worker nodes.
+
+To organize your distributed training runs:
+
+1. Set `job_type` for different node types to categorize your nodes based on their function:
+
+   ```python
+   # Main coordinating node
+   wandb.init(
+       project="distributed-training",
+       group="experiment_1",
+       job_type="main"
+   )
+
+   # Reporting worker nodes
+   wandb.init(
+       project="distributed-training", 
+       group="experiment_1",
+       job_type="worker"
+   )
+   ```
+1. Create filtered views in your workspace to organize your runs:
```

Comment:
What is meant by "filtered view"? Is it like a saved workspace? If yes, link to docs the describe this.